.github/workflows/aead.yml

```diff
@@ -34,6 +34,8 @@ jobs:
         with:
           toolchain: ${{ matrix.rust }}
           targets: ${{ matrix.target }}
+      # Isolate this crate from workspace which is otherwise MSRV 1.60 due to namespaced features
+      - run: rm ../Cargo.toml
```

Comment:
Instead of this you could invert the `exclude` in Cargo.toml, placing all of the 1.60+ crates in `members`, and the <1.60 crates in `exclude`:

https://github.com/RustCrypto/traits/blob/master/Cargo.toml#L15